src/main/java/core/basesyntax/service/strategy/SupplyOperation.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.service.strategy;
+
+import core.basesyntax.dao.FruitStorageDao;
+import core.basesyntax.model.FruitTransaction;
+
+public class SupplyOperation implements OperationHandler {
+    private FruitStorageDao storageDao;
```

Comment:
Following the encapsulation principle, consider changing the visibility of `storageDao` to private if it is not intended to be accessed directly from outside the class.